src/plugins/dashboard/public/dashboard_container/embeddable/create/create_dashboard.ts

```diff
@@ -487,7 +460,7 @@ export const initializeDashboard = async ({
       startWith(dashboardContainer.getInput()),
       map((input) => input.filters),
       distinctUntilChanged((previous, current) => {
-        return compareFilters(previous ?? [], current ?? []);
+        return compareFilters(previous ?? [], current ?? [], COMPARE_ALL_OPTIONS);
```

Comment:
default `comparatorOptions` is, which means that compareOptions will return true (no changes) when there are any changes in those keys. Which is not what we want. 
```
comparators = defaults(comparatorOptions || {}, {
    index: false,
    state: false,
    negate: false,
    disabled: false,
    alias: false,
  });
```

Reply:
This makes sense because here we're comparing filters as state, not determining whether or not to fetch data. 👍